lib/cdo/contact_rollups.rb

```diff
@@ -25,32 +30,13 @@ class ContactRollups
     query_timeout: MAX_EXECUTION_TIME_SEC
   )
 
-  # Connection to read from Pegasus reporting database.
-  PEGASUS_REPORTING_DB_READER = sequel_connect(
-    CDO.pegasus_reporting_db_reader,
-    CDO.pegasus_reporting_db_reader,
-    query_timeout: MAX_EXECUTION_TIME_SEC
-  )
-
-  # Connection to write to Pegasus reporting database.
-  PEGASUS_REPORTING_DB_WRITER = sequel_connect(
-    CDO.pegasus_reporting_db_writer,
-    CDO.pegasus_reporting_db_writer,
-    query_timeout: MAX_EXECUTION_TIME_SEC
-  )
-
-  # Connection to read from Dashboard reporting database.
-  DASHBOARD_REPORTING_DB_READER = sequel_connect(
-    CDO.dashboard_reporting_db_reader,
-    CDO.dashboard_reporting_db_reader,
-    query_timeout: MAX_EXECUTION_TIME_SEC
-  )
-
   def self.mysql_multi_connection
     # return a connection with the MULTI_STATEMENTS flag set that allows multiple statements in one DB call
+    pegasus_clone_writer_uri = URI(CDO.pegasus_reporting_db_writer)
+    pegasus_clone_writer_uri.host = @@clone_db_endpoint
```

Comment:
Passing state through class variables makes for code that's hard to reason about and test in isolation, I'd recommend refactoring this to a parameter so the flow control is more clear.